design/baremetal-operator/uefi-http-boot.md

```diff
@@ -0,0 +1,102 @@
+# Add UEFI HTTP boot hardware driver
+
+## Status
+
+Implementable
+
+## Summary
+
+This design proposes to add a new hardware driver `redfish-http-boot` to
```

Comment:
I"m slightly inclined to omit the word "boot" and use something like `redfish-uefihttp` for consistency with both the existing `redfish-virtualmedia` and what Ironic has.

Reply:
that makes sense, I'll change it